web/src/App.js

```diff
@@ -1,19 +1,30 @@
 import React from 'react';
+import './App.scss';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import Tab from 'react-bootstrap/Tab';
 import Tabs from 'react-bootstrap/Tabs';
+import TopNav from './components/TopNav';
+import Footer from './components/Footer';
 import Lookup from './components/Lookup';
 import Alerts from './components/Alerts';
 import Registration from './components/Registration';
 import VerifyContact from './components/VerifyContact';
 import Route from './components/nav/Route';
 import NoRoute from './components/nav/NoRoute';
 import MultiRoute from './components/nav/MultiRoute';
+import { isTestNet } from './lib/testnet';
 
+const testNet = isTestNet();
 const mainRoutes = new Set(["/", "/register"]);
 const allRoutes = new Set(["/", "/register", "/verify-contact"]);
 
+// Unchanging constants that should be configured when the react app is built.
+const headerClassName = testNet ? "bg-testnet-gradient" : "bg-gradient";
+const headerTitle = testNet ? "TRISA TestNet Directory" : "TRISA Global Directory Service";
+const headerLead = testNet ? "Get started with the TRISA TestNet to implement your Travel Rule compliance service." : "Become a TRISA certified Virtual Asset Service Provider.";
+const switchLink = testNet ? <a href="https://vaspdirectory.net">Production Directory Service</a> : <a href="https://trisatest.net">TestNet Directory Service</a>
```

Comment:
These constants are so ugly, but I can't think of another mechanism for this :-( 

I tried to keep things in constants that are computed at when the app starts so it's not constantly recomputing these strings.